inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
++
+_Note_: We recommend working on the latest version always. So check the
+website once in a while and download the new .jar-file when a later
+version is available.
+** Now, simply ***double-click ***on the downloaded .jar-file. After a
+moment, a splash screen will display while the application is
+initializing. Once the initialization is complete, a dialog appears.
+Here, you can open the application in your default browser or shut it
+down again.
+** If you are used to the command line, you may enter this command there
+instead:
++
+$ java -jar inception-app-standalone-0.10.0.jar
++
+In this case, no splash screen will start. Just enter
+http://localhost:8080/[_http://localhost:8080_] in you browser.
+** Use „admin“ as username and password to login, since a default user
+with these properties is created the first time you start the
+application.
 
-The first time you start the application, a default user with the name `admin` and the password
-`admin` is created. Use this username and password to log in to the application after opening it
-in your browser.
+* Download an example project for your first steps
+** Also on our website, clicking the tab „Example Projects“ on the very
+top leads you to the following page:
+https://inception-project.github.io/example-projects/ . To follow this
+Guide, please download the _Interactive Concept Linking_ project.
++
 
-== Your first annotation
+First Steps with INCEpTION
 
-After logging in, you will see the main menu. Click on *Projects* to go to the 
-*project management page* and there click on *create* to start a new project. Enter a name
-for your project, select the type *annotation project* and press *save*.
+* You may want to watch our tutorial video „Overview“ on how to create
+annotation:
+https://www.youtube.com/watch?v=wp4AN3p23mQ&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&index=3&t=0s
+* In order to import our example project, after logging into INCEpTION,
+click on the blue folder on the top right (next to „Import project
+archives“) and browse for the example project you have downloaded
+before. Finally, click the grey **upload-**Butten right next to the blue
+folder to upload the selected project.
 
-Next, switch to the *Documents* tab and press *choose files* to select a plain text file from
-your local harddisk (the file should be in UTF-8 encoding). As part of the import process, 
-{product-name} automatically processes the file to identify sentence and token boundaries.
+* The project has now been added. Choose it by clicking on it and you
+will be on the dashboard of this project. Select *Annotation* on the top
+left and choose _pets1.tsv_ (the first file) in the window which now
+opened.
+* You will see several annotations already. If you import a documents
+without annotations yet, there will be none. But in the example
+projects, we have added some annotations already to show you some
+examples of annotations.
+* To create your first annotation, let‘s start with Part of Speech
```

Comment:
Why do you use the concept linking project for pos and ner?
You also never explained layers before and what pos and ner is

Reply:
POS is explained right there. Layers right after.

I do not explain NER in depth because we do not use it at all. Explaing auxillary concepts in order to understand this guide becomes cumbersome. I mentioned the basics before, in the paragraph starting with "To follow this Guide, please download the Interactive Concep...." ( I think after you write this comment ? ).

I use this project because
- it does not matter what kind of annotation to make in order to learn the mechanism how to do it.
- we do not have a POS-Project
- Not everybody knows Concept Linking. POS is something everybody knows the concept of who passed elementry

In order to not confuse people, I mentioned now that the project is for NER (and what it is) but we can do any kinds of annotations and start with POS since t is simple

Which project and layer to you think is better for a first annotation?

Reply:
P.S: I might swich to NE layer instead of POS. But I find, the relation between NER/NE/NEL/concept linking is not obvious. People might wonder why we use a NE layer in a concept linking project just as much as for a POS layer in a concept linking project. Explaining the whole context goes way to far in my opinion... 
What do you suggest?

Reply:
I swiched to NE now in order to avoid confusion. I'll give further background knowledge in one of the boxes we talked about to explain why we use a NE layer in a concept linking project.